test/performance/perf_test.py

```diff
@@ -83,12 +81,18 @@ def tearDown(self):
         print('Running %s. MEDIAN=%s' % (self.__class__.__name__,
                                          self.percentile(50)))
         result_data.append({
-            'name': name,
-            'results': {
-                '1': {
-                    'ops_per_sec': result
-                }
-            }
+            'info': {
+                'test_name': name,
+                'args': {
+                   'threads': 1,
+                },
+            },
+            'metrics': [
+                {
+                    'name': 'ops_per_sec',
```

Comment:
So, ops_per_sec has always been used because that was the only option, but IIRC that's not what our metrics actually are. Is it possible to fix that now?

Reply:
Oh I see, the current result is more like "bytes of data processed per second". We can actually add multiple stats now, like can can add the median latency in addition to a bytes/sec number. Any suggestions for what to call the new field? Maybe "bytes_per_sec"?

Reply:
If that's what it is (it's been so long I don't remember) than LGTM.